gen3-cli/cot/backend/common/context.py

```diff
@@ -65,6 +65,11 @@ def cwd(self, value):
     def root(self):
         return self.__root
 
+    # Cache dir path
+    @property
+    def cache(self):
```

Comment:
it'd rather name it `cache_dir` just to be more clear

Reply:
Ye, you're right. It's confusing. I'll change that.